installation_tips/README.md

```diff
@@ -1,36 +1,53 @@
 ## Installation tips
 
-If you are not (yet) an expert in Python installations (conda vs pip, mananging environements, etc.),
-here we propose a simple recipe to install `spikeinterface` and several sorters inside an anaconda
-environment for windows/mac users.
+If you are not (yet) an expert in Python installations, a main difficulty is choosing the installation procedure.
+The main ideas you need to know before starting:
+ * python itself can be distributed and installed many many ways.
+ * python itself do not contain so many features for scientifique computing you need to install "packages".
+   Numpy, scipy, matplotlib, spikeinterface, ... are python packages that have a complicated dependency graph between then. "uv"
+ * installing package  can be distributed and installed several ways (pip, conda, uv, mamba, ...)
+ * installing many packages at once is challenging (because of the depenency graph) so you need to do it in an "isolated environement"
+   to not destroy any previous installation. You need to see an "environement" as a sub installtion in dedicated folder.
+
+Choosing the installator + a environement manager + a package installer is a nightmare for beginners.
+The main options are:
+  * use "anaconda" that do everything. The most popular but bad idea because : ultra slow and agressive licensing (not free anymore)
```

Comment:
```suggestion
  * use "anaconda", which does everything. The most popular but bad idea because : ultra slow and aggressive licensing (not always free anymore)
```